scheduler/src/cook/mesos/schema.clj

```diff
@@ -443,7 +443,9 @@
              - task failed, no other tasks, retries remaining => job waiting
              - task failed, other tasks running => job running
 
-             Note that backfilled running instances are treated as if they don't exist."
+             Note that backfilled running instances are treated as if they don't exist.
+             In other words, if a job has running tasks that are all backfilled, the
```

Comment:
@wyegelwel   I don't think we decided that?   Here's the convo about that:  https://github.com/twosigma/Cook/pull/145#discussion-diff-69572273  We speculated about different things that could be done to simplify the states, but no definitive conclusion.